package utils

func CheckError(err error) {
	panic(err)
}
